src/libraries/ErrorsLib.sol

```diff
@@ -64,7 +64,7 @@ library ErrorsLib {
     error MarketNotCreated();
 
     /// @notice Thrown when submitting a non previously enabled market for removal.
-    error MarketNotEnabled();
+    error MarketNotEnabled(Id id);
```

Comment:
Let's change also the natspec to
```
/// @notice Thrown when interacting with a non previously enabled market `id`.
```
Notice 2 changes: 
- added `id` to the natspec
- the error is not just emitted on forced market removals